### PR TITLE
[hailctl] Get ready for deployment!

### PR DESCRIPTION
**Pull request by tpoterba**
Made the following changes:

 - Disabled dataproc tests
 - Moved dataproc tests to Makefile, to be run before manual deploys
 - Add back VEP cluster test script
 - Removed cloudtools config files
 - Removed the latest-build functionality
 - Added VEP scripts to hailctl/dataproc/resources
 - Changed init_notebook to pip install hail wheels, picking up
   dependencies automatically
 - add out-of-date check (once per day) to hailctl startup

### CODE REVIEW

**Comment by tpoterba** [comment]
I had a late-breaking inspiration to remove the phony tags, you're totally right.

Also I fixed some paths in the init scripts so the notebooks work now. I'll address the rest of your comments now.

**Comment by tpoterba** [comment]
as you can tell, I am not good with `make`.

File: hail/Makefile [hunk 1]
```diff
@@ -66,26 +66,51 @@ wheel: build-info shadowJar init-scripts
...
 	echo "dataproc:" >> python/hailctl/deploy.yaml
-	$(foreach FILENAME, $(RESOURCES_TO_COPY), \
-	    gsutil cp python/hailctl/dataproc/resources/$(FILENAME) $(CLOUD_LOC); \
-	    gsutil acl set public-read $(CLOUD_LOC); \
-	    echo "  $(FILENAME): $(CLOUD_LOC)" >> python/hailctl/deploy.yaml;)
+	for FILE in `ls python/hailctl/dataproc/resources`; do \
+	  CLOUD_LOC=$(HAILCTL_BUCKET_BASE)/$(DEV_CLARIFIER)/$(CLOUD_SUB_FOLDER)/$$FILE && \
+	  gsutil -m cp python/hailctl/dataproc/resources/$$FILE $$CLOUD_LOC && \
+	  gsutil acl set public-read $$CLOUD_LOC && \
+	  echo "  $$FILE: $$CLOUD_LOC" >> python/hailctl/deploy.yaml || break; done || exit
```
[the remark this answers is not in the document]
Comment (tpoterba) [reply hail/Makefile]: dunno, that would be fine too. Do you mind if I leave it?

[the remark this answers is not in the document]
Comment (danking) [reply hail/Makefile]: all good

File: hail/Makefile [hunk 2]
```diff
@@ -66,26 +67,56 @@ wheel: build-info shadowJar init-scripts
...
+	gsutil -m cp $(wheel_path) $(wheel_cloud_path)
+	gsutil acl set public-read $(wheel_cloud_path)
+
+.PHONY: install-wheel
+install-wheel: wheel
+	pip uninstall -y hail
+	pip install $(wheel_path)
 
-.PHONY: pypi-deploy
+.PHONY: install-hailctl-wheel
```
Comment (danking) [inline hail/Makefile]: I really like this style of PHONY next to the rule in question, but it, oddly, seems non-standard. ❤️

Comment (tpoterba) [reply hail/Makefile]: I copied this style from some stuff I saw on SO and also in the autogenerated sphinx makefile. It does seem much clearer -- the clarifier is next to the code in question.

**Comment by tpoterba** [comment]
Thanks for the review. Next step is a bit of documentation.

**Pull request merged**